api/v1alpha1/networkfence_types.go

```diff
@@ -46,6 +56,12 @@ type NetworkFenceSpec struct {
 	// +kubebuilder:validation:Required
 	Driver string `json:"driver"`
 
+	// FenceState contains the desired state for the CIDRs
+	// mentioned in the Spec. i.e. Fenced or Unfenced
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Enum=Fenced;Unfenced
```

Comment:
won't this be a backward compatibility issue for existing CRs?

Reply:
set `kubebuilder:default:Fenced` so that we dont have any issues for the existing CR's.

Reply:
ACK